### PR TITLE
mdd-seo-category-chart

### PR DESCRIPTION
**Pull request by mddifilippo89**
column-chart-with-highlighting

### CODE REVIEW

File: samples/charts/category-chart/column-chart-with-highlighting/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+import './index.css';
+import { IgrCategoryChart } from 'igniteui-react-charts';
+import { IgrCategoryChartModule } from 'igniteui-react-charts';
+
+IgrCategoryChartModule.register();
+
+export default class CategoryChartColumnChartWithHighlighting extends React.Component<any, any> {
+    public data: any[];
+    public chart: IgrCategoryChart;
+    public excludedProperties: string[] = ["NY_Low", "LA_Low"];
```
Comment (HUSSAR-mtrela) [inline samples/charts/category-chart/column-chart-with-highlighting/src/index.tsx:12]: remove this field

File: samples/charts/category-chart/column-chart-with-highlighting/src/index.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+                    onChange={this.onItemHighlightingChanged}/>Item </label>
+                    <label className="options-label"><input type="checkbox"
+                    checked={this.state.isCategoryHighlighting}
+                    onChange={this.onCategoryHighlightingChanged}/>Category </label>
+                </div>
+            </div>
+
+            <div className="container" style={{height: "calc(100% - 30px)"}} >
+                <IgrCategoryChart
+                    excludedProperties ={this.excludedProperties}
```
Comment (HUSSAR-mtrela) [inline samples/charts/category-chart/column-chart-with-highlighting/src/index.tsx:46]: remove this setting

File: samples/charts/category-chart/column-chart-with-highlighting/src/index.tsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+            { Month : "March", NY_High : 12.2, NY_Low : -3.8, LA_High : 27.8, LA_Low : 8.3 },
+            { Month : "April", NY_High : 11.7, NY_Low : 2.2, LA_High : 33.9, LA_Low : 10.6 },
+            { Month : "May", NY_High : 19.4, NY_Low : 1.1, LA_High : 35.0, LA_Low : 13.9 },
+            { Month : "June", NY_High : 23.3, NY_Low : 10.6, LA_High : 36.7, LA_Low : 16.1 },
+            { Month : "July", NY_High : 27.2, NY_Low : 19.4, LA_High : 33.3, LA_Low : 15.6 },
+            { Month : "August", NY_High : 25.6, NY_Low : 16.7, LA_High : 36.7, LA_Low : 15.6 },
+            { Month : "September", NY_High : 22.8, NY_Low : 8.9, LA_High : 43.9, LA_Low : 16.1 },
+            { Month : "October", NY_High : 17.8, NY_Low : 0.0, LA_High : 38.3, LA_Low : 11.1 },
+            { Month : "November", NY_High : 17.8, NY_Low : -1, LA_High : 32.8, LA_Low : 6.7 },
+            { Month : "December", NY_High : 8.3, NY_Low : -6.6, LA_High : 28.9, LA_Low : 5.6 },
```
Comment (HUSSAR-mtrela) [inline samples/charts/category-chart/column-chart-with-highlighting/src/index.tsx:83]: remove "NY_Low", "LA_Low" data columns and
rename "NY_High" to "NY", rename "LA_High" to "LA"

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]

File: samples/charts/category-chart/column-chart-with-highlighting/src/index.tsx (hunk 1 above)
Comment (mddifilippo89) [reply samples/charts/category-chart/column-chart-with-highlighting/src/index.tsx:12]: done

File: samples/charts/category-chart/column-chart-with-highlighting/src/index.tsx (hunk 2 above)
Comment (mddifilippo89) [reply samples/charts/category-chart/column-chart-with-highlighting/src/index.tsx:46]: done

File: samples/charts/category-chart/column-chart-with-highlighting/src/index.tsx (hunk 3 above)
Comment (mddifilippo89) [reply samples/charts/category-chart/column-chart-with-highlighting/src/index.tsx]: done

**Review by HUSSAR-mtrela — APPROVED** [review]

**Pull request merged**